### PR TITLE
Added dotnet-fody.

### PR DESCRIPTION
**Pull request by droyad**
Implements #206

Documentation: https://github.com/droyad/Fody-Wiki/blob/master/NetCore.md

The nupkg is currently being built by the AfterBuild task and put into `dotnet-fody/`.

### CODE REVIEW

File: dotnet-fody.Tests/ResponseFileParserTests.cs [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+--reference:C:\Users\bob\.nuget\packages\Microsoft.CSharp\4.0.1\ref\netstandard1.0\Microsoft.CSharp.dll
+--reference:C:\Users\bob\.nuget\packages\Microsoft.NETCore.Portable.Compatibility\1.0.1\ref\netstandard1.0\System.ComponentModel.DataAnnotations.dll
+C:\source\MyProject\src\TestApp\Program.cs
+C:\source\MyProject\src\TestApp\Properties\AssemblyInfo.cs";
+
+        [Test]
+        public void IntermediateDirectory()
+        {
+            Assert.AreEqual(
+                @"C:\source\MyProject\src\TestApp\obj\Debug\netcoreapp1.0\",
```
Comment (SimonCropp) [inline dotnet-fody.Tests/ResponseFileParserTests.cs]: wont this fail on other people machines?

File: dotnet-fody/App.config [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (SimonCropp) [inline dotnet-fody/App.config]: delete this file

File: dotnet-fody.Tests/dotnet-fody.Tests.csproj [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    <ProjectReference Include="..\dotnet-fody\dotnet-fody.csproj">
+      <Project>{2DD93D98-DEC1-44EB-9F9D-C871E90FCFCB}</Project>
+      <Name>dotnet-fody</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Properties\" />
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <!-- To modify your build process, add your task inside one of the targets below and uncomment it.
```
Comment (SimonCropp) [inline dotnet-fody.Tests/dotnet-fody.Tests.csproj]: remove the comment

File: dotnet-fody.Tests/dotnet-fody.Tests.csproj [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    <None Include="packages.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\dotnet-fody\dotnet-fody.csproj">
+      <Project>{2DD93D98-DEC1-44EB-9F9D-C871E90FCFCB}</Project>
+      <Name>dotnet-fody</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Properties\" />
```
Comment (SimonCropp) [inline dotnet-fody.Tests/dotnet-fody.Tests.csproj]: remove the redundant properties

File: dotnet-fody.Tests/ResponseFileParserTests.cs (hunk 1 above)
Comment (droyad) [inline dotnet-fody.Tests/ResponseFileParserTests.cs]: No, because it is parsing the string constant defined above.

**Comment by NPSF3000** [comment]
What's the status of this?

**Comment by SimonCropp** [comment]
given fody now supports netstandard i think we can close this?

**Comment by ysiw** [comment]
@SimonCropp Fody doesn't work with dotnet/cli. 
```cmd
H:\work\c#\hello>type hello.csproj
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp1.1</TargetFramework>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Fody" Version="2.0.6" />
    <PackageReference Include="PropertyChanged.Fody" Version="1.53.0" />
  </ItemGroup>

</Project>

H:\work\c#\hello>dotnet restore
  Restoring packages for H:\work\c#\hello\hello.csproj...
  Generating MSBuild file H:\work\c#\hello\obj\hello.csproj.nuget.g.props.
  Generating MSBuild file H:\work\c#\hello\obj\hello.csproj.nuget.g.targets.
  Writing lock file to disk. Path: H:\work\c#\hello\obj\project.assets.json
  Restore completed in 1.26 sec for H:\work\c#\hello\hello.csproj.

  NuGet Config files used:
      C:\Users\ysiw\AppData\Roaming\NuGet\NuGet.Config
      C:\Program Files (x86)\NuGet\Config\Microsoft.VisualStudio.Offline.config

  Feeds used:
      https://api.nuget.org/v3/index.json
      C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\

H:\work\c#\hello>dotnet build
Microsoft (R) Build Engine version 15.1.548.43366
Copyright (C) Microsoft Corporation. All rights reserved.

C:\Users\ysiw\.nuget\packages\fody\2.0.6\build\netstandard1.4\Fody.targets(53,5): error MSB4062: The "Fody.WeavingTask" task could not be loaded from the assembly C:\Users\ysiw\.nuget\packages\fody\2.0.6\build\netstandard1.4\..\..\\Fody.dll. Could not load file or assembly 'Microsoft.Build.Utilities.v4.0, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. ç³»ç»Ÿæ‰¾ä¸åˆ°æŒ‡å®šçš„æ–‡ä»¶ã€‚ Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask. [H:\work\c#\hello\hello.csproj]

Build FAILED.

C:\Users\ysiw\.nuget\packages\fody\2.0.6\build\netstandard1.4\Fody.targets(53,5): error MSB4062: The "Fody.WeavingTask" task could not be loaded from the assembly C:\Users\ysiw\.nuget\packages\fody\2.0.6\build\netstandard1.4\..\..\\Fody.dll. Could not load file or assembly 'Microsoft.Build.Utilities.v4.0, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. ç³»ç»Ÿæ‰¾ä¸åˆ°æŒ‡å®šçš„æ–‡ä»¶ã€‚ Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask. [H:\work\c#\hello\hello.csproj]
    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:02.26

H:\work\c#\hello>dotnet

Microsoft .NET Core Shared Framework Host

  Version  : 1.1.0
  Build    : 928f77c4bc3f49d892459992fb6e1d5542cb5e86

Usage: dotnet [common-options] [[options] path-to-application]

Common Options:
  --help                           Display .NET Core Shared Framework Host help.
  --version                        Display .NET Core Shared Framework Host version.

Options:
  --fx-version <version>           Version of the installed Shared Framework to use to run the application.
  --additionalprobingpath <path>   Path containing probing policy and assemblies to probe for.

Path to Application:
  The path to a .NET Core managed application, dll or exe file to execute.

If you are debugging the Shared Framework Host, set 'COREHOST_TRACE' to '1' in your environment.

To get started on developing applications for .NET Core, install the SDK from:
  http://go.microsoft.com/fwlink/?LinkID=798306&clcid=0x409

H:\work\c#\hello>
```

**Comment by droyad** [comment]
Closing this as the new build system has superceeded the need for this

**Pull request closed without merging**